src/api/init.ts

```diff
@@ -123,7 +128,18 @@ export async function startApiServer(datastore: DataStore, chainId: ChainID): Pr
   );
 
   // Setup direct proxy to core-node RPC endpoints (/v2)
-  app.use('/v2', createCoreNodeRpcProxyRouter());
+  // pricing endpoint
+  app.use(
+    '/v2',
+    (() => {
+      const router = addAsync(express.Router());
+      router.use(cors());
+      router.use('/prices', createBnsPriceRouter(datastore, chainId));
```

Comment:
@asimm241 @noumanahmads I just noticed the `docs/openapi.yaml` file is missing definitions for routes under `/v2/prices/`, but the json schema files exist

Reply:
Fixed in e1805ec